src/QUMainWindow.ui

```diff
@@ -60,7 +60,7 @@
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
-         <enum>Qt::CustomContextMenu</enum>
+         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>

```

Comment:
Where do all these changes come from/what do they do? Since they don't seem to change any functionality

Reply:
I *believe* these changes come automatically from an updated Designer which reflects changes in Qt? I'd assume that `Qt::CustomContextMenu` was moved to `Qt::ContextMenuPolicy::CustomContextMenu`? I had to edit the ui file (in Designer) to change the swisscharts label and the name of the checkbox. Did that and saved, noting more.

Reply:
This seems a bit arbitrary, but if that's what the designer does... well